internal/controller/deployment_to_pdb_controller.go

```diff
@@ -186,9 +222,27 @@ func (r *DeploymentToPDBReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&v1.Deployment{}).
 		WithEventFilter(predicate.Funcs{
+			CreateFunc: func(e event.CreateEvent) bool {
```

Comment:
not what I meant just put the if around the entire return ctrl.NewControllerManagedBy(mgr).